src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString
+    .split(';')
```

Comment:
[CODE STYLE] - The semicolon splitting and subsequent operations can be chained directly without storing them in an intermediate variable `stylesArray`. This would make the code more concise and easier to follow.